src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py

```diff
@@ -97,8 +97,6 @@ def checkJobStateTransition(jobID, candidateState, currentStatus=None, jobMonito
     """Utility to check if a job state transition is allowed"""
     if not currentStatus:
         if not jobMonitoringClient:
-            from DIRAC.WorkloadManagementSystem.Client.JobMonitoringClient import JobMonitoringClient
```

Comment:
Circular import is no longer an issue? (I think that was the reason for this construct?)

Reply:
Good question. I don't know if there is still an issue there, but I revert it nonetheless.